modules/backend/clinic/staff/receptionist/ReceptionistMock.cpp

```diff
@@ -0,0 +1 @@
+#include "ReceptionistMock.hpp"
```

Comment:
Will some functionality be placed in mock cpp file? (I don't know what is the plan)
If not we could stick to hpp and then create cpp if needed.

Reply:
For code measures/analysis tools (which I hope we will add at some point) it is necessary to have both hpp and cpp files as far as I know.

Reply:
Ok, I didn't know. I did not use any such tools directly. I don't know how they work actually.
So  let's keep it. We can remove them anytime we want if needed.
If we manage to move mocks to tests module, this will matter even less, as tests module can live it own rules and we can do anything what makes sense to us.